embedded-hal-bus/src/spi/exclusive.rs

```diff
@@ -49,6 +49,15 @@ impl<BUS, CS, D> ExclusiveDevice<BUS, CS, D> {
     }
 }
 
+use unwrap_infallible::UnwrapInfallible;
+
+impl<BUS, D> ExclusiveDevice<BUS, NoPin, D> {
+    /// Create a new [`ExclusiveDevice`] without a Chip Select (CS) pin.
+    pub fn new_no_cs(bus: BUS, delay: D) -> Self {
+        ExclusiveDevice::new(bus, NoPin, delay).unwrap_infallible()
```

Comment:
To not pull another dependency, you can also do

```rust
pub fn new_no_cs(bus: BUS, delay: D) -> Self {
    match ExclusiveDevice::new(bus, NoPin, delay) {
        Ok(this) => this,
        // no Err() branch
    }
}
```

Less readable, but avoids dependency-bloat...  Not sure what the official stance of e-h on that topic is...

Reply:
yeah avoiding deps is always nice.